src/js/components/search/table/ResultsTable.jsx

```diff
@@ -96,7 +96,6 @@ export default class ResultsTable extends React.PureComponent {
         const columns = this.props.columns.map((column, i) => {
```

Comment:
The ResultsTable is actually designed to be able to return different cell components for different columns (see note in `ResultsTableGenericCell`). You'll want to add some logic in here to check if the column is the ID column and then return the custom ID cell instead of the generic cell in that case.